ext/dom/element.c

```diff
@@ -1251,8 +1251,7 @@ PHP_METHOD(DOMElement, replaceWith)
 	id = ZEND_THIS;
 	DOM_GET_OBJ(context, id, xmlNodePtr, intern);
 
-	dom_parent_node_after(intern, args, argc);
-	dom_child_node_remove(intern);
+	dom_child_replace_with(intern, args, argc);
```

Comment:
Unrelated, but the ``argc`` variable should be of type ``uint32_t``

Reply:
Yeah, something for a follow-up.